x-pack/plugins/observability/public/pages/cases/helpers.ts

```diff
@@ -4,16 +4,91 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { useEffect, useState } from 'react';
+import { useEffect, useState, useCallback, useMemo } from 'react';
 import { isEmpty } from 'lodash';
+
+import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
 import { usePluginContext } from '../../hooks/use_plugin_context';
 import { TopAlert, parseAlert } from '../../pages/alerts/';
 import { useKibana } from '../../utils/kibana_react';
-import { Ecs } from '../../../../cases/common';
+import { BASE_RAC_ALERTS_API_PATH } from '../../../../rule_registry/common/constants';
+
+export const useFetchAlertData = (alertIds: string[]): [boolean, Record<string, unknown>] => {
+  const { http } = useKibana().services;
```

Comment:
tip: you can also use `useHttp()` it returns the http object directly.

Reply:
Are you referring to this one? https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/common/lib/kibana/hooks.ts#L32

Doesn't look like I can access that while inside of observability, should I move the hook int `cases/common`?

![image](https://user-images.githubusercontent.com/56361221/151587076-1d44d783-1093-4da2-94d8-d09b01de55b3.png)

Reply:
I do not think you should move it to cases/common so other plugins can use it. `useHttp` is a cases helper for internal use and `useKibana` is more appropriate in other plugins (unless they have their own helpers).